cmd/stellar-rpc/internal/preflight/preflight_test.go

```diff
@@ -98,24 +98,41 @@ var mockLedgerEntriesWithoutTTLs = []xdr.LedgerEntry{
 		Data: xdr.LedgerEntryData{
 			Type: xdr.LedgerEntryTypeConfigSetting,
 			ConfigSetting: &xdr.ConfigSettingEntry{
-				ConfigSettingId: xdr.ConfigSettingIdConfigSettingContractLedgerCostV0,
+				ConfigSettingId:      xdr.ConfigSettingIdConfigSettingContractLedgerCostV0,
+				ContractMaxSizeBytes: nil,
+				ContractCompute:      nil,
 				ContractLedgerCost: &xdr.ConfigSettingContractLedgerCostV0{
-					LedgerMaxReadLedgerEntries:     100,
-					LedgerMaxReadBytes:             100,
-					LedgerMaxWriteLedgerEntries:    100,
-					LedgerMaxWriteBytes:            100,
-					TxMaxReadLedgerEntries:         100,
-					TxMaxReadBytes:                 100,
-					TxMaxWriteLedgerEntries:        100,
-					TxMaxWriteBytes:                100,
-					FeeReadLedgerEntry:             100,
-					FeeWriteLedgerEntry:            100,
-					FeeRead1Kb:                     100,
-					BucketListTargetSizeBytes:      100,
-					WriteFee1KbBucketListLow:       1,
-					WriteFee1KbBucketListHigh:      1,
-					BucketListWriteFeeGrowthFactor: 1,
+					// TODO(fons): adjust
```

Comment:
Is this TODO for this PR or later?

Reply:
Ahh, I should address that before merging. Thanks